window.c

```diff
@@ -139,6 +140,7 @@ void win_init(win_t *win)
 	INIT_ATOM_(_NET_WM_ICON);
 	INIT_ATOM_(_NET_WM_STATE);
 	INIT_ATOM_(_NET_WM_STATE_FULLSCREEN);
+	INIT_ATOM_(_NET_WM_PID);
```

Comment:
Might be a good idea to keep this in the same order as the list in `sxiv.h` line 387. Not necessarily important, but improves consistency.

```suggestion
	INIT_ATOM_(_NET_WM_PID);
	INIT_ATOM_(_NET_WM_STATE_FULLSCREEN);
```